node/subsystem-types/src/messages.rs

```diff
@@ -592,6 +592,10 @@ pub enum ChainSelectionMessage {
 	/// Request the best leaf containing the given block in its ancestry. Return `None` if
 	/// there is no such leaf.
 	BestLeafContaining(Hash, oneshot::Sender<Option<Hash>>),
+	/// Passed block number and hash are for the relay parent of the disputed candidate.
+	/// The parent must be marked as reverted, and its children must be marked as
+	/// non-viable.
```

Comment:
I don't understand this. Why we revert the parent of a disputed candidate? Isn't just the block containing the candidate and all its children reverted?

Reply:
My mistake, relay parent is exactly the relay chain block containing the candidate. I got confused.